extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzRuntimeConfig.java

```diff
@@ -19,6 +19,17 @@ public class QuartzRuntimeConfig {
     @ConfigItem(defaultValue = "QuarkusQuartzScheduler")
     public String instanceName;
 
+    /**
```

Comment:
Let's also allow configuring `acquireTriggersWithinLock`. This config item has to default to `true`? 

This property has to be if JDBC JobStore is used i.e anywhere here https://github.com/quarkusio/quarkus/blob/19bded9c48d6d65fb4615b46e1d80d8f74930d57/extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java#L577-L587

Reply:
Yes the acquireTriggersWithinLock has to be true if JDBC JobStore is used, but the default value is either false or true. As of Quartz 2.1.1 “true is the default if batchTriggerAquisitionMaxCount is set to >1 otherwise false by defualt.